### ISSUE TITLE
Hi.

### ISSUE THREAD

**Issue by NullArray** [jesuiscamille/AutoSQLi#16]
Hi! Sorry for cluttering up your ticket feed. Your code base has changed a lot since last i checked it, I was wondering are you still incorporating DorkNet or features derived from DorkNet? And if so, if you made some cool improvements, consider opening a Pull Request. At my repo: 

https://github.com/NullArray/DorkNet

Of course you will be accredited for any contributions. Acknowledgements are important. 

Also;

> When I woke up this morning, someone said that a guy by the name of NullArray tweeted about this project. 11 stars later, it makes me want to finish it more than ever !

I'm actually (VectorSEC)[https://twitter.com/Real__Vector] on Twitter. I know, confusing.  If you want we could make this the `General Discussion Thread` like we had with (AutoSploit)[https://github.com/NullArray/AutoSploit] 

Lemme know. See you on Discord.

**Comment by NullArray** [comment]
True, DorkNet is relatively slow since it basically emulates a browser. Also,i'm not sure whether i spoke to you on Twitter about DorkNet but i remember having a conversation about the amount of results. Like i said then, DorkNet was primarily designed to be used with a list of dorks, so 10 dorks, with 20 results per page works out to 200 results even though i am only processing the first page of results per Dork, I will check Googler out, since it sounds interesting.

I guess it makes use of the Google API?

By the way, when do you expect to have a functional BETA for this project? Pretty interested to see how it will turn out. Keep up the good work :+1:

**Comment by jesuiscamille** [comment]
It wasn't me since I'm not on twitter.
The tool which was used before ( sqlidumper, which I think isn't open source ) had approximatively 10 working sql injection for 10 000 target urls. If I want to get such a number of urls with DorkNet as now, I need to create 2 * 10 000 / 10 requests, so 2 000 requests to google. I think it might ban an ip for a while :p.

I think that Googler makes use of the text browser available ( such as lynx ): [ref](https://github.com/jarun/googler/blob/7691f305cf1f1aa155e512fd39401e2a489bc0a1/googler#L89).

I think it'll be functional in two or three weeks, and pretty stable maybe in a month or two. In fact, if I reach a beta state I would be pretty happy :)

**Comment by NullArray** [comment]
Cool, good to know. Once you have a functional beta, let me know. I'll check it out, if it works well, i'll post a little bit about your tool on my Twitter feed, if that's alright with you. Will certainly help with exposure for your project (n_n")
